restapi/stories.go

```diff
@@ -57,12 +73,74 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 
 	for i := 0; i < len(stories); i++ {
 		city := stories[i].CurrentCity
-		coordinates, err := api.locationFinder.GetCityCenter(city)
+		stories[i].CurrentCity = strings.Title(strings.ToLower(city))
+
+		// get number of stories with current city in db
+		var prevStories []models.Story
+		err := api.database.Where("current_city=?", city).Find(&prevStories).Error
+		if err != nil {
+			api.logger.Infof("A")
```

Comment:
More descriptive log message? (Ditto with stuff below it)

Reply:
Also when logging errors please use `logger.Fatalw("...")`

Reply:
got rid of all the logs